rdt/transformers/id.py

```diff
@@ -0,0 +1,62 @@
+"""Transformers for id data."""
+import pandas as pd
+
+from rdt.transformers.base import BaseTransformer
+
+
+class IDGenerator(BaseTransformer):
+    """Generate an ID column.
+
+    This transformer generates an ID column based on a given prefix, starting value and suffix.
+
+    Args:
+            prefix (str):
+                Prefix of the generated IDs column.
+                Defaults to ``None``.
+            starting_value (int):
+                Starting value of the generated IDs column.
+                Defaults to ``0``.
+            suffix (str):
+                Suffix of the generated IDs column.
+                Defaults to ``None``.
+    """
+
+    INPUT_SDTYPE = 'id'
+
+    def __init__(self, prefix=None, starting_value=0, suffix=None):
+        super().__init__()
+        self.prefix = prefix
+        self.starting_value = starting_value
+        self.suffix = suffix
+        self._counter = 0
+        self.output_properties = {None: {'next_transformer': None}}
+
+    def reset_sampling(self):
```

Comment:
Should this be `reset_randomization` instead? Otherwise it won't get called automatically on re-fit.

Reply:
Yes good catch thanks! Done in [33d4deb](https://github.com/sdv-dev/RDT/pull/680/commits/33d4deb7971abfdb186744e517981c2399f88d43)